docs/modules/ROOT/pages/gsn-bouncers.adoc

```diff
@@ -10,110 +10,50 @@ If you're still learning about the basics of the Gas Station Network, you should
 
-The actual user's `msg.sender` and `msg.data` can be obtained safely via `_msgSender()` and `_msgData()` of https://docs.openzeppelin.com/contracts/2.x/api/gsn#GSNRecipient[`GSNRecipient`].
+The actual user's `msg.sender` and `msg.data` can be obtained safely via xref:api:gsn.adoc#GSNRecipient-_msgSender--[`_msgSender()`] and xref:api:gsn.adoc#GSNRecipient-_msgData--[`_msgData()`] of xref:api:gsn.adoc#GSNRecipient[`GSNRecipient`].
 
-Deciding how to approve and reject transactions is a bit more complex. The GSN recipient contract, by default, will accept and pay for all transactions. Chances are you probably want to choose which users can use your contracts via the GSN and potentially charge them for it, like a bouncer at a nightclub. We call these contracts _GSN Bouncers_.
+Deciding how to approve and reject relayed calls is a bit more complex. The GSN recipient contract, with the simplest implementation, will accept and pay for all relayed calls. Chances are you probably want to choose which users can use your contracts via the GSN and potentially charge them for it, like a bouncer at a nightclub. We call these contracts _GSN Bouncers_.
 
-We include `GSNBouncerSignature` and `GSNBouncerERC20Fee`, ready to use out of the box, along with how to create your own Custom Bouncer.
+In this guide we describe how to use the included bouncers xref:api:gsn.adoc#GSNBouncerSignature[`GSNBouncerSignature`] and xref:api:gsn.adoc#GSNBouncerERC20Fee[`GSNBouncerERC20Fee`], along with how to create your own Custom Bouncer.
 
 == GSNBouncerSignature
 
-The https://docs.openzeppelin.com/contracts/2.x/api/gsn#GSNBouncerSignature[`GSNBouncerSignature`] lets users call your recipient contract via the GSN (charging you for it) if they can prove that an account you trust approved them to do so. The way they do this is via a _signature_.
+xref:api:gsn.adoc#GSNBouncerSignature[`GSNBouncerSignature`] lets users relay calls via the GSN to your recipient contract (charging you for it) if they can prove that an account you trust approved them to do so. The way they do this is via a _signature_.
 
-The transaction request must be signed by the same account that was added to the contract as a trusted signer. If it is not the same, the `GSNBouncerSignature` will not accept the transaction.
+The relayed call must include a signature by the same account that was added to the contract as a trusted signer. If it is not the same, `GSNBouncerSignature` will not accept the relayed call.
 
-This means that you need to set up a system where your trusted account signs transaction requests, as long as they are valid users (according to your business logic).
+This means that you need to set up a system where your trusted account includes their signature with the relayed call, as long as they are valid users (according to your business logic).
```

Comment:
I think this may lead people to think that this same system itself relays the call. It's more like the system sends the signature back to the user, who then relays it off to the GSN. Sorry if I confused you about this with my previous review.

Reply:
I'll try adding wording more like Dennison's which should hopefully be clearer

Reply:
Updated with new wording.